vault/tests/conftest.py

```diff
@@ -77,10 +77,11 @@ def __call__(self):
             user = getpass.getuser()
             chown_args = ['chown', user, self.token_file]
 
-            if user != 'root':
```

Comment:
maybe extract as run_command_as_sudo and apply to haproxy too?

Reply:
If you refer to just the `sudo` prefix part, I'm not sure it worth it the short version is just: `run_command(["sudo"] + chown_args, check=True)`

If you refer to the try/except. We can maybe create a `run_command_and_try_with_sudo_if_fail` kind of util. But not sure if it worth it if we only have two cases and will only save few lines :)

Reply:
Actually for haproxy, sudo seems to be needed locally and on CI.